src/main/kotlin/com/grepp/nbe1_3_team04/chat/api/request/ChatRequest.kt

```diff
@@ -0,0 +1,15 @@
+package com.grepp.nbe1_3_team04.chat.api.request
+
+import com.grepp.nbe1_3_team04.chat.service.request.ChatServiceRequest
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.NotNull
+
+@JvmRecord
+data class ChatRequest(
+    val chatroomId: @NotNull(message = "채팅방 아이디는 필수입니다.") Long?,
+    val message: @NotBlank(message = "채팅 메세지는 필수입니다.") String?
+) {
+    fun toServiceRequest(): ChatServiceRequest {
+        return ChatServiceRequest(chatroomId!!, message!!)
```

Comment:
저도 DTO 좀 찾아보면서 알아낸 내용인데, 민우님 코드는 지난번에 봐서 알지만 제가 검색한 내용도 공유만 하려고 합니다.

```suggestion
        requireNotNull(chatroomId) { "채팅방 아이디는 null이 될 수 없습니다." }
        requireNotNull(message) { "채팅 메세지는 null이 될 수 없습니다." }
        return ChatServiceRequest(chatroomId, message)
```

이미 위쪽에서도 비슷한 내용으로 다루신 것 같은데, 이런 식으로 체크하는 방식도 있더라구요.
이건 그냥 공부한 내용 공유라고 생각해주세요!

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/require-not-null.html

Reply:
넵 좋은 방식인 것 같습니다!

다만 검증 어노테이션으로 값이 절때 null이 아니라는게 이미 검증이 됐다면 `!!` 연산자를 쓰는게 더 깔끔하다고 생각합니다!